utils/callData/registrationCalls.ts

```diff
@@ -125,16 +141,52 @@ function multiCallRenewal(
   });
 }
 
+function multiCallAutoRenewal(
+  domains: string[],
+  metahash: string,
+  salesTaxRate: number
+): Call[] {
+  let calls = [
+    {
+      contractAddress: process.env.NEXT_PUBLIC_ETHER_CONTRACT as string,
+      entrypoint: "approve",
+      calldata: [
+        process.env.NEXT_PUBLIC_RENEWAL_CONTRACT as string,
+        "340282366920938463463374607431768211455",
+        "340282366920938463463374607431768211455",
+      ],
+    },
+  ];
+
+  domains.map((domain) => {
```

Comment:
I think we shouldn't check that here because we never did with the others (encoding, and pricing computation)